multistep/commonsteps/step_create_cdrom.go

```diff
@@ -291,3 +302,18 @@ func (s *StepCreateCD) AddFile(dst, src string) error {
 
 	return filepath.Walk(src, visit)
 }
+
+func (s *StepCreateCD) AddContent(dst, path, content string) error {
+	// Join Cleans the path so we can join it without path traversal issues.
+	dstPath := filepath.Join(dst, path)
```

Comment:
I removed the `filepath.Clean` call as `Join` already does that ! :)

Reply:
@azr From testing, doing `filepath.Join("/folder", "../hello")` would result in `/hello` which I don't think would be intended behaviour. Doing `filepath.Clean` on an absolute path would turn `../hello` into `/hello` so it joins to `/folder/hello`.